<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlay2Class</class>
 <widget class="QWidget" name="XPlay2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlay2</string>
  </property>
  <widget class="XVideoWidget" name="video">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>620</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QSlider" name="playPos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>1221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPause">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>620</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>XVideoWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="XPlay2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>XPlay2Class</receiver>
   <slot>OpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>860</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>865</x>
     <y>794</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenFile()</slot>
 </slots>
</ui>
